src/Cli - Ex 3/cliTodoFunctions.js

```diff
@@ -0,0 +1,158 @@
+import chalk from 'chalk';
+import inquirer from 'inquirer';
+import fs from 'fs';
+import PokemonClient from './PokemonClient.js';
+import imageToAscii from 'image-to-ascii';
+
+let rawdata = fs.readFileSync('./todolist.json');
+let tasks = JSON.parse(rawdata);
+const Pokemon = new PokemonClient();
+
+class cliTodoFunctions {
+	constructor(tasks) {
+		this.tasks = tasks;
+	}
+
+	async getPokemon(id) {
+		const pokemonName = await Pokemon.getPokemon(id);
+		return pokemonName;
+	}
+
+	writeToList(tasksTobeAdded = tasks) {
```

Comment:
why do you need that default you can use this.tasks

Reply:
this is pretty confusing naming for the param, you get all of the tasks and its support also removing

Reply:
one last thing we usually mark private methods with _ (_writeToList) so it would be clear which are used from outside of this file and which are not